src/strings/strings-en.ts

```diff
@@ -878,6 +885,8 @@ export const strings = {
   STORING: 'Storing',
   STORING_START_DATE: 'Storing Start Date',
   STRATIFICATION: 'Stratification',
+  SUB_LOCATION_DETAILS: 'Sub-Location Details',
+  SUB_LOCATION_EXISTS: 'This storage location already exists',
```

Comment:
Should this use "sub-location" instead of "storage location"?

Reply:
Oops, yes, good catch. Will update.